### ISSUE TITLE
Request: an option to run `yay -Pww` before any update

### ISSUE THREAD

**Issue by getzze** [Jguer/yay#1751]
Duplicate of #384 that was closed because it became stale.

I will repeat here the last argument in https://github.com/Jguer/yay/issues/384#issuecomment-770110776 
- Having an option to show news before any updating command seems helpful, especially for newbies, as one is not always aware of problem due to packages needing manual intervention. And it would be easier to have this option than creating, for instance, an alias.

That's very annoying and it struck me again today because of the [wireplumer problem](https://archlinux.org/news/undone-replacement-of-pipewire-media-session-with-wireplumber/).

Thanks for considering implementing it!

**Comment by emilyst** [comment]
As an alternative, consider [informant](https://aur.archlinux.org/packages/informant). It installs a Pacman hook.

**Comment by freedg** [comment]
Adapted informant's hook to use yay:

```desktop
[Trigger]
Operation = Upgrade
Type = Package
Target = *
Target = !yay

[Action]
Description = Checking Arch News with yay ...
When = PreTransaction
Exec = /usr/bin/sudo -u nobody -g nobody /usr/bin/yay -Pw
```
Putting this into `/etc/pacman.d/hooks/00-yay-news.hook` works well enough. Of course `-Pw` can be changed to `-Pww` if you want that. Permissions are reduced with `sudo -u nobody -g nobody`, otherwise yay complains about being ran as root. 
I took `AbortOnFail` out, as yay does not exit non-zero when there is new news. I think it would be a good idea to add an additional arg like `-!` or `--loud` to have yay exit non-zero when there is new news.

**Comment by getzze** [comment]
I have been using `informant` since @emilyst  mentioned it (thanks!). It helps a lot but I am not very satisfied because the hook is triggered too late, just before starting the Update phase, meaning that yay first downloads all the packages, update the AUR package PKGBUILDs and then fails. It's not totally lost time as the downloaded packages remain in cache, but it's very annoying to start a long update, come back later and find out that it errored halfway.
The best timing for me would be to call yay -Pw (or informant or whatever) just after pulling the list of new versions (so you know if you will be affected by some changes).
